test-i2c/test-i2c.ino

```diff
@@ -0,0 +1,144 @@
+ //imports
```

Comment:
- you should copy the contents of `IMU_V2.ino` into this file and delete `IMU_V2.ino`,  `.ino` files should be in a folder with the same name as them